pipeline/runtime/runtest.go

```diff
@@ -91,10 +91,14 @@ func executeRunTestStep(ctx context.Context, f RunFunc, r *api.StartStepRequest,
 	}
 	exportEnvs, _ := fetchExportedVarsFromEnvFile(exportEnvFile, out, useCINewGodotEnvVersion)
 	artifact, _ := fetchArtifactDataFromArtifactFile(artifactFile, out)
-	if len(r.Outputs) > 0 {
```

Comment:
Why changes are needed in runtest ? run.go should be used for plugins right ?

Reply:
this is just a refactor of logic to match the format of run.go and runtest.go for easier readability .

Reply:
I agree, we can skip adding in runtest.
The long term goal is for test intelligence to just be a plugin and to remove run tests special implementation from here.